content/client-server-api/_index.md

```diff
@@ -2927,7 +2927,7 @@ that profile.
 | [Typing Notifications](#typing-notifications)              | Required  | Required | Required | Required | Optional |
 | [User and Room Mentions](#user-and-room-mentions)          | Required  | Required | Required | Optional | Optional |
 | [Voice over IP](#voice-over-ip)                            | Required  | Required | Required | Optional | Optional |
-| [Client Config](#client-config)                            | Optional  | Optional | Optional | Optional | Optional |
+| [Account Data](#client-config)                            | Optional  | Optional | Optional | Optional | Optional |
```

Comment:
You also need to apply changes to the URL fragment. `#client-config` should have been changed to `#account-data`, here and maybe in other parts of the spec.